src/docfx/lib/JsonUtility.cs

```diff
@@ -184,6 +184,12 @@ public static JObject Merge(JObject first, IEnumerable<JObject> objs)
             return result;
         }
 
+        public static JObject ValidateNullValue(this JObject graph)
+        {
+            var (_, jtoken) = ((JToken)graph).ValidateNullValue();
```

Comment:
>ValidateNullValue [](start = 46, length = 17)

We need to remove null silently in LegacyMarkdown. Will this method log error if finding null?